cpp/src/strings/copying/concatenate.cu

```diff
@@ -121,7 +121,7 @@ __global__ void fused_concatenate_string_offset_kernel(column_device_view const*
                                                        bitmask_type* output_mask,
                                                        size_type* out_valid_count)
 {
-  size_type output_index     = threadIdx.x + blockIdx.x * blockDim.x;
+  int64_t output_index       = threadIdx.x + blockIdx.x * blockDim.x;
```

Comment:
Would you please use the `cudf::thread_index_type` alias?

Reply:
Thanks @GregoryKimball , addressed that.